test/cri-containerd/containerdrestart_test.go

```diff
@@ -0,0 +1,69 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"testing"
+
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+// CRI will terminate any running containers when it is restarted.
+// Run a container, restart containerd, validate the container is terminated.
+func Test_ContainerdRestart_LCOW(t *testing.T) {
+	requireFeatures(t, featureLCOW)
+
+	pullRequiredLCOWImages(t, []string{imageLcowK8sPause, imageLcowAlpine})
+
+	client := newTestRuntimeClient(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	sandboxRequest := getRunPodSandboxRequest(t, lcowRuntimeHandler, nil)
```

Comment:
this function also changed and now accepts `sandboxOpts ...SandboxConfigOpt` instead of annotation map, so `nil` should be removed here.

Reply:
Fixed!